src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Confirm.tsx

```diff
@@ -570,7 +570,7 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
     const txid = await buildAndBroadcast({
       adapter,
       buildCustomTxInput,
-      receiverAddress: undefined, // no receiver for this contract call
+      receiverAddress: await adapter.getAddress({ accountNumber, wallet }),
```

Comment:
This should never be the next receive address

```suggestion
      receiverAddress: maybeFromUTXOAccountAddress,
```

Reply:
nice catch!

Reply:
tackled in https://github.com/shapeshift/web/pull/5842/commits/8221fa8805a04cbbbdde1318bcc09a0b3873acfb